base_tier_validation/templates/tier_validation_templates.xml

```diff
@@ -94,6 +94,7 @@
             <field name="sequence" />
             <field name="requested_by" />
             <field name="status" />
```

Comment:
Did not func test it. But maybe only display the field `display_status` here.

Reply:
The field will be needed for the conditional class. https://github.com/OCA/server-ux/pull/830/files#diff-8183ea04ccd831a8c08ad90be8130df46e9d358a6b4f330a36f93cd348a51b3cR70